sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -100,4 +100,20 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <profiles>
+    <profile>
+      <id>java-lts</id>
+      <activation>
+        <jdk>[11,)</jdk>
```

Comment:
This activation profile can be scoped more by adding in a check for environment variable `AZURE_TEST_HTTP_CLIENTS` being `jdk`. This may be the better way to go, because in that scenario we could potentially remove the Netty implementation from the class path, which would allow us to fully use Maven to handle which HTTP client will be used and we no longer need to hook this logic into JUnit as we do now.

Reply:
https://maven.apache.org/surefire/maven-surefire-plugin/examples/configuring-classpath.html

Reply:
@srnagar @anuchandy @JonathanGiles Thoughts on this logic change

Reply:
We will have a full matrix of tests enabled for all clients and all service versions that run less frequently than the nightly live tests. So, this profile should be activated when the env variable is set to either `jdk` or `all`. So, we should not scope it down to just `jdk`. And when it's `all`, letting maven handle classpath is going to be tricky.